scripts/release-notes/gen_ec_release_notes.py

```diff
@@ -404,6 +407,32 @@ def tag_exists(release_name):
         return False
 
 
+def add_token_remote():
+    """
+    Adds the Git remote to the given repository using
+    the provided installation (or personal) access token.
+    """
+    env = {}
+    env.update(os.environ)
+
+    print(f'git remote remove {REMOTE}')
+    subprocess.run(
+       ["git", "remote", "remove", REMOTE],
+       env=env,
+    )
+    print(f'git remote add {REMOTE} ~~REDACTED~~')
+    remote_url = f"https://x-access-token:{GITHUB_TOKEN}@github.com/{GITHUB_ORG}/{GITHUB_REPO}"
+    result = subprocess.run(
+       ["git", "remote", "add", REMOTE, remote_url],
+       env=env,
+       capture_output=True,
+       text=True,
+    )
+    if result.returncode != 0:
+        err = result.stderr.replace(GITHUB_TOKEN, "") if result.stderr else "stderr is empty"
+        raise Exception(f"Command `git remote add` failed: {err}")
```

Comment:
You could use CalledProcessError, like on line 459, but I wouldn't hold this up over it.

Reply:
I noticed it, but didn't quite fit here - I didn't want to put `cmd` in there and redact two things